interface/patient_file/summary/immunizations.php

```diff
@@ -343,497 +346,474 @@ function saveImmunizationObservationResults($id, $immunizationdata)
-                          value="<?php echo ($id != 0 && $value['imo_criteria'] == 'vaccine_type') ? attr($value['imo_code']) : ''; ?>" style="width:140px;" />
-        <div class="imm-imm-add-12" id="imm-imm-add-123"><?php echo ($id != 0 && $value['imo_criteria'] == 'vaccine_type') ? text($value['imo_codetext']) : ''; ?></div>
-                          <input type="hidden"  value="CVX" name="code_type_hidden[]" id="code_type_hidden3"/>
-        <input type="hidden" class="code_text_hidden" name="code_text_hidden[]" id="code_text_hidden3" value="<?php echo ($id != 0 && $value['imo_criteria'] == 'vaccine_type') ? attr($value['imo_codetext']) : ''; ?>"/>
-                        </td>
-       <td <?php echo ($value['imo_criteria'] != 'vaccine_type' || $id == 0) ? 'style="display: none;"' : ''; ?> class="vis_published_date_td" id="vis_published_date_td_1">
-        <label><?php echo xlt('Date VIS Published'); ?></label>
+                            if ($entered_by) { ?>
+                                <p><?php echo xlt('Entered By'); ?> <?php echo text($entered_by); ?></p>
+                            <?php } ?>
+
+                            <button type="button" class="btn btn-primary" onclick="showObservationResultSection();" title='<?php echo xla('Click here to see observation results'); ?>'>
+                                <?php echo xlt('See observation results'); ?>
+                            </button>
+                        </div>
+                    </div>
+
+                    <div class="observation_results" style="display:none;">
+                        <fieldset class="obs_res_head">
+                            <legend><?php echo xlt('Observation Results'); ?></legend>
+                            <div class="obs_res_table">
+                                <?php
+                                if (!empty($imm_obs_data) && count($imm_obs_data) > 0) {
+                                    foreach ($imm_obs_data as $key => $value) {
+                                        $key_snomed = 0;
+                                        $key_cvx = 0;
+                                        $style = '';?>
+                                        <div class="form-row" id="or_tr_<?php echo attr(($key + 1)); ?>">
+                                            <?php
+                                            if ($id == 0) {
+                                                if ($key == 0) {
+                                                    $style = 'display: table-cell;width:765px !important';
+                                                } else {
+                                                    $style = 'display: none;width:765px !important';
+                                                }
+                                            } else {
+                                                $style = 'display : table-cell;width:765px !important';
```

Comment:
Change the display CSS to a class -- probably will have to rework php to do this

Reply:
I am not sure, but I try to figure out, it seems like can remove.

cc @bradymiller

Reply:
You don't have to do this part -- i'll just fix it later